common/views/components/Header/Header.js

```diff
@@ -7,16 +8,17 @@ type Props = {|
   links: Array<{ href: string, title: string }>
 |}
 
-const Header = ({ links, siteSection }: Props) => (
-  <div className='header grid js-header-burger js-focus-trap bg-white'>
+const Header = withToggler(({ links, siteSection, toggle, isActive }: Props) => (
+  <div className={`header grid js-header-burger js-focus-trap bg-white ${isActive ? 'header--is-burger-open' : ''}`}>
     <span className='visually-hidden js-trap-reverse-end'>reset focus</span>
     <div className='header__upper grid__cell'>
       <div className='header__inner container'>
         <div className='header__burger'>
           <a href='#footer-nav-1'
             id='header-burger-trigger'
             className='header__burger-trigger js-header-burger-trigger js-trap-start'
-            aria-label='menu'>
+            aria-label='menu'
+            onClick={(event) => { event.preventDefault(); toggle(event); }}>
```

Comment:
Should this `toggle` be receiving the `event` as an argument? It doesn't look like it's a parameter [here](https://github.com/wellcometrust/wellcomecollection.org/blob/87525570e4dce2ba8e462325516aa6a6ac3bdf3b/common/views/hocs/withToggler.js#L14). Or am I missing something?

Reply:
Nope, you're catching things... things Flow should catch:
https://flow.org/blog/2017/05/07/Strict-Function-Call-Arity/#calling-a-function